docs/dev/table/connect.md

```diff
@@ -652,6 +686,34 @@ connector:
   path: "file:///path/to/whatever"    # required: path to a file or directory
 {% endhighlight %}
 </div>
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- CREATE a partitioned CSV table using the CREATE TABLE syntax.
+create table csv_table (
+  user bigint,
+  message string,
+  ts string
+) 
+COMMENT 'This is a csv table.' 
+PARTITIONED BY(user)
+WITH (
+  'connector.type' = 'filesystem',  -- required: specify to connector type
```

Comment:
I think we only need to list `connector.type` and `connector.path` just like YAML and Java/Scala.
Other properties are not the part of FileSystem.

Reply:
I don't think so, at lease we should give out the required properties like `format.type`.

Reply:
I agree with Jark, we should not duplicate code too much. You could have some surrounding skeleton but then focus on the important part:
```
CREATE TABLE (...) WITH (
   'connector.type' = 'filesystem'
)
```